listenbrainz/feedback.py

```diff
@@ -0,0 +1,24 @@
+import uuid
+
+from typing import List
+from pydantic import BaseModel, ValidationError, validator
+
+
+class Feedback(BaseModel):
```

Comment:
should be some basic documentation here explaining what a feedback is.
@paramsingh what do you think about location of this? We could either keep it in the module in db associated with reading/writing them, or make a separate `models` package. Thoughts?

Reply:
Documentation added.

Reply:
hmm, i'm thinking of how we'll import it and I think for now it makes sense to keep in the db module.